packages/store-ui/src/atoms/Radio/Radio.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import Radio from './Radio'
+
+describe('Radio', () => {
```

Comment:
Can we test if the click event is working fine? Or we can't because it's a Radio?

Reply:
I think we do not need that.